stories/Input/form.js

```diff
@@ -265,7 +265,7 @@ storiesOf('Inputs', module)
           label="Phone number"
           mask="111-111-111"
           placeholder="Type your phone number"
-          type="text"
+          type="phone"
```

Comment:
it shouldn't be tel?
```suggestion
          type="tel"
```

Reply:
Nope, this is one case where we are showing the change from `phone`  to `tel`.

Reply:
I think its a bit weird, `tel` attribute is the standard for phone inputs in the web, why not keep as the same?

Reply:
The idea is to make it retrocompatible with our old and wrong "phone" `type`, and this story here is testing if we are correctly converting `phone` to `tel` in the component's implementation.

I believe the idea is to maintain both `phone` and `tel` on our `type` prop at least for the time being :slightly_smiling_face: